src/core/common/instance.hpp

```diff
@@ -862,6 +862,13 @@ template <> inline BorderRouter::RoutingManager &Instance::Get(void)
 }
 #endif
 
+#if OPENTHREAD_CONFIG_SRP_SERVER_ENABLE
+template <> inline Srp::SrpServer &Instance::Get(void)
```

Comment:
Suggest renaming to `Srp::Server` instead of `Srp::SrpServer`.

This harmonizes it with pattern used by other net protocol implementations in OT: `Dns::Clinet`, `Dhcp6::Client`, `Dhcp6::Server`, and `Sntp::Client`.